api/v1alpha1/nginxingresscontroller_types.go

```diff
@@ -61,6 +61,10 @@ type DefaultSSLCertificate struct {
 	// Secret is a struct that holds the name and namespace fields used for the default ssl secret
```

Comment:
we need cel validation guaranteeing that only one of secret and keyvaulturi are filled out